### PR TITLE
cards, list groups, and tabs

### PR DESCRIPTION
**Pull request by andrewmcodes**
## Type of PR

- [x] feature
- [ ] enhancement
- [ ] bug fix
- [ ] other

## Description

- Creates cards, list group, and tabs components.
- Removes websites controller and resources since it's unused.

## Checklist

- [x] My code follows the style guidelines of this project & the tests are passing
- [x] I have performed a self-review of my own code
- [x] I have requested a review from another developer(s)

### CODE REVIEW

**Comment by andrewmcodes** [comment]
Not sure why system tests are failing in CI - they pass locally. This happens occasionally for some reason.

File: app/components/avatar_component.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class AvatarComponent < ViewComponent::Base
```
Comment (coderberry) [inline app/components/avatar_component.rb:1]: This prevents me from setting a specific size for the gravatar. Also, if we are using the gravatar, do we want to pass in the size (converted to px) in the `user_gravatar_url` call?

File: app/components/card_component.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    @class_names = classes
+  end
+
+  private
+
+  attr_reader :class_names
+
+  def classes
+    classes = ["card"]
+    classes.push(class_names) unless class_names.try(:empty?)
```
Comment (coderberry) [inline app/components/card_component.rb:12]:
```suggestion
    classes.push(class_names) unless class_names.empty?
```

Comment (coderberry) [reply app/components/card_component.rb:12]: I think you can call .empty? on nil right?

Comment (andrewmcodes) [reply app/components/card_component.rb:12]: You cannot :)

File: app/components/avatar_component.rb (hunk 1 above)
Comment (andrewmcodes) [reply app/components/avatar_component.rb:1]: We are already using this today, I just took it out of the module it was in. If you’d like to look into that implementation, we should do it in another PR

File: app/components/card_component.rb (hunk 2 above)
Comment (andrewmcodes) [reply app/components/card_component.rb:12]: But I can probably refactor this now that I look at it

**Pull request merged**